.gitignore

```diff
@@ -0,0 +1,2 @@
+/nbproject/
+/heron/ux/oleditor/ole/client/
```

Comment:
I don't think this file should be part of the PR. It is your local preference. Pls remove.